core/db.py

```
@@ -294,3 +322,121 @@ async def init_db_async() -> None:
         await conn.run_sync(metadata.create_all)
+
+
+def get_db_connection():
+    """Get a database connection for testing purposes.
+
+    RU: Возвращает соединение c базой данных для тестирования.
+    EN: Returns a database connection for testing purposes.
+    """
+    try:
+        return _RAW_ENGINE.connect()
+    except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)